### ISSUE TITLE
Weird tab order in Launcher

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#3616]
Tab order (i.e. the order in which the items are focused when pressing tab) should be from top to bottom, but now it's all over the place. See video below (the order in which the items are given the blue highlight)

https://github.com/user-attachments/assets/2ed247a8-a7db-4301-a6a2-de08b627b81e
